packages/api/src/moderation/subjects/feed-generator.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  ModerationSubjectFeedGenerator,
+  ModerationDecision,
+  ModerationOpts,
+} from '../types'
+
+export function decideFeedGenerator(
+  subject: ModerationSubjectFeedGenerator,
+  opts: ModerationOpts,
+): ModerationDecision {
+  // TODO handle labels applied on the feed generator itself
```

Comment:
Just confirming this is intended to be left as a TODO for now!

Reply:
yep!